{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "security@odoo.com",
      "ID": "CVE-2021-45111"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Improper access control in Odoo Community 15.0 and earlier and Odoo Enterprise 15.0 and earlier allows remote authenticated users to trigger the creation of demonstration data, including user accounts with known credentials."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/odoo/odoo/issues/107683",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/odoo/odoo/issues/107683"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-04-25T22:07Z",
  "publishedDate": "2023-04-25T19:15Z"
}